alertmanager/receiver.go

```diff
@@ -0,0 +1,256 @@
+// promconfig
+// Copyright 2020 Percona LLC
+//
+// Based on Prometheus systems and service monitoring server.
+// Copyright 2015 The Prometheus Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alertmanager
+
+import (
+	"time"
+
+	"github.com/percona/promconfig"
+)
+
+// Receiver configuration provides configuration on how to contact a receiver.
```

Comment:
wasn't sure whether we wanted all of these notifier types or not, let me know if they are not needed.

Reply:
We don't plan to support them all in the first release, but you already copied them, and that's not a problem :) Let's keep them